news/2022-09-27-changes-to-osu-taiko-sr-and-pp.md

```diff
@@ -24,31 +24,30 @@ We will also be using terminology coined by the osu!taiko community to refer to
 
-## An insight into the base of the new rework
+## An insight into the basis of the new rework
 
-The last set of changes made to osu!taiko's performance points & star rating algorithm aimed to chart several skills at once. This not only made it difficult to collect meaningful and targeted feedback relevant to one specific skill, but also led to increased complexity which halted continuous development efforts. 
+  The [previous set of changes](https://osu.ppy.sh/home/news/2020-09-15-changes-to-osutaiko-star-rating) made to osu!taiko's performance points & star rating algorithm accounted for several skill areas at once. This made it difficult to collect meaningful feedback on specific skills and led to increased complexity, halting forward development efforts. 
 
-As such, we have opted to take a more modular approach by introducing new changes incrementally, in the hopes of being able to fix any reported issues one step at a time. Moving forward, osu!taiko star rating will be receiving continuous updates, potentially placing star rating and performance points in a state of flux, especially over the coming few months. 
+This time, we have opted to take a more modular approach by introducing new changes incrementally, in the hopes of being able to fix any reported issues one step at a time. Moving forward, osu!taiko star rating will be receiving continuous updates, placing star rating and performance points in a state of flux, especially over the coming few months. 
 
-During this time, we will be working on other difficulty components (both small and large-scale) in preparation for future releases. It is our hope that this will continue to help the algorithm reflect the latest trends and expectation in difficulty calculation, but we also apologise in any advance for any disruption this may cause as we begin working things out.
 
-In the star rating part of this rework, we have essentially completely reworked the stamina and colour skills, introducing new concepts and the infrastructure required to support them.
+In the star rating part of this rework, we have focused on "stamina" and "colour" skills.
 
 Performance points have seen significant changes to mod multipliers and a broad change to how accuracy is calculated. Please note however that these changes are *experimental* and did not receive as much scrutiny as the rest of the rework. Consider leaving feedback if you find anything that feels especially unusual.
 
 Without further ado, let's get into the nitty-gritty of these changes.
 
 ## Star rating changes
 
-### Readdressing stamina
+### "Stamina" skill
```

Comment:
This should probably have added a "The" before, to match the header below (or remove the "The" below, whatever feels better to you).